components/Accordion/react/Accordion.tsx

```diff
@@ -29,22 +77,88 @@ export const Accordion: React.FC<
   titleClassName,
   descriptionClassName,
   fullWidthContent,
+  open,
+  locked = false,
+  onClickSummary,
+  onToggle = () => {},
+  onToggleBlocked = () => {},
```

Comment:
I'm not sure we should be setting default empty funcs for these props

Reply:
They are optional.

Reply:
ok, I'll make that change.